### ISSUE TITLE
why sregistry[-cli] 0.0.5  is needed to build and install?

### ISSUE THREAD

**Issue by Trophime** [singularityhub/sregistry#106]
Hi,
I've noticed in `requirements.txt` needed for installing `sregistry` that you are
using `sregistry[-cli] 0.0.5` from `PiPy`.

I'm wondering why you don't use a more recent version of  `sregistry[-cli]` ?
and why `sregistry-cli` is actually needed?

Trying, very naively to update cli to a recent version, I end up with an error:
```
uwsgi_1   | Traceback (most recent call last):
uwsgi_1   |   File "shub/wsgi.py", line 7, in <module>
uwsgi_1   |     application = get_wsgi_application()
uwsgi_1   |   File "/usr/local/lib/python3.5/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
uwsgi_1   |     django.setup(set_prefix=False)
uwsgi_1   |   File "/usr/local/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
uwsgi_1   |     apps.populate(settings.INSTALLED_APPS)
uwsgi_1   |   File "/usr/local/lib/python3.5/site-packages/django/apps/registry.py", line 108, in populate
uwsgi_1   |     app_config.import_models()
uwsgi_1   |   File "/usr/local/lib/python3.5/site-packages/django/apps/config.py", line 202, in import_models
uwsgi_1   |     self.models_module = import_module(models_module_name)
uwsgi_1   |   File "/usr/local/lib/python3.5/importlib/__init__.py", line 126, in import_module
uwsgi_1   |     return _bootstrap._gcd_import(name[level:], package, level)
uwsgi_1   |   File "<frozen importlib._bootstrap>", line 986, in _gcd_import
uwsgi_1   |   File "<frozen importlib._bootstrap>", line 969, in _find_and_load
uwsgi_1   |   File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
uwsgi_1   |   File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
uwsgi_1   |   File "<frozen importlib._bootstrap_external>", line 662, in exec_module
uwsgi_1   |   File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
uwsgi_1   |   File "./shub/apps/api/models.py", line 24, in <module>
uwsgi_1   |     from shub.apps.api.actions import create_container
uwsgi_1   |   File "./shub/apps/api/actions/__init__.py", line 24, in <module>
uwsgi_1   |     from .delete import delete_container
uwsgi_1   |   File "./shub/apps/api/actions/delete.py", line 23, in <module>
uwsgi_1   |     from sregistry.auth import generate_timestamp
uwsgi_1   | ImportError: cannot import name 'generate_timestamp'
```

**Comment by vsoch** [comment]
oh this is a good catch! The specification of the version to begin with is to prevent errors like this, and the reason it's not updated is just forgetting on my part :) The reason that the blank install doesn't work is because after that version I fine tuned the client install so dependencies would be closely matched with the client that needed them. This meant that the functions for authentication (and pertaining to the timestamp) were moved into the registry module (because we only need them here!) and so the updated install that should do the trick is to update the requirements.txt, but also update the import location of those functions! I'll open a PR asap with these changes for you.

**Comment by vsoch** [comment]
@Trophime if you didn't see, I fixed this in #107 and when you can give a thumbs up, we can merge. Thanks for catching this! I would have completely forgotten to update the version.

**Comment by vsoch** [comment]
This is merged and fixed, thanks!
